packages/crowdstrike/data_stream/falcon/_dev/test/pipeline/test-falcon-tags-list.log-expected.json

```
@@ -0,0 +1,72 @@
+{
+    "expected": [
+        {
+            "@timestamp": "2022-09-12T19:08:08.832Z",
+            "cloud": {
+                "account": {
+                    "id": "XXXXXXXXXXXX"
+                },
+                "provider": "AWS",
+                "region": "us-west-2",
+                "service": {
+                    "name": "EC2"
+                }
+            },
+            "crowdstrike": {
+                "event": {
+                    "Finding": "Instance ID: i-0108fce80e5ab5129|VPC ID: vpc-0e886040c27d9f526|Network ACL ID: acl-005e6bb98e75ac17e|Rule Number: 100|CIDR Block: 0.0.0.0/0|Protocol: All",
+                    "PolicyId": 26,
+                    "ResourceAttributes": {
+                        " ACL ID": "acl - 005e6 bb98e75ac17e",
```

```suggestion
                        "ACL ID": "acl - 005e6 bb98e75ac17e",
```

I think it is just in the [test input](https://github.com/elastic/integrations/pull/7625/files#diff-d59dcbf2fafc17dd0bdf402eca886ed6527d6829a4901987d730b1b3ebda17d2R26)